sdk/identity/azure-identity/out/azure/identity/_credentials/browser.pyi

```diff
@@ -0,0 +1,11 @@
+# Stubs for azure.identity._credentials.browser (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._internal import AuthCodeRedirectServer, PublicClientCredential, wrap_exceptions
+from azure.core.credentials import AccessToken
+from typing import Any
+
+class InteractiveBrowserCredential(PublicClientCredential):
+    def __init__(self, client_id: str, **kwargs: Any) -> None: ...
```

Comment:
client_id should be optional here and moved to kwargs

Reply:
Defaulting to Azure CLI?